### DISCUSSION TITLE
今天更新后ipad上无法使用

### DISCUSSION THREAD

**Discussion by THMonster** [imfuxiao/Hamster#257] [General]
能进app，但切输入法的菜单里也能看到仓输入法，但切不过去。

**Comment by imfuxiao** [comment]
非常抱歉，精力有限，目前主要在处理iPhone上的版本，iPad都没咋关注。

等过段时间，iPhone功能完善的差不多了，就来处理iPad的问题。

**Comment by g6ai** [comment]
欸，我用着没问题，今日版本加最新iOS

**Reply by THMonster** [reply]
你是ios还是ipados？

**Reply by g6ai** [reply]
是iPadOS 16.5，设备是iPad Pro 11寸（2018）

**Comment by THMonster** [comment]
重启ipad后正常了
